Packs/AccentureCTI/IndicatorTypes/indicatortype-ACTI_Malware_Family.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
Would like to see usage of those indicator types and fields. Mostly to know what was missing from the current fields for you